src/components/ToDoComponent/SettingGoals/CreateGoals.tsx

```diff
@@ -68,96 +108,118 @@ const MainScreen = ({ navigation }) => {
           placeholderTextColor="red"
         />
-        <Text style={styles.titleText}>Description</Text>
+        {titleError ? <Text style={styles.error}>{titleError}</Text> : null}
+
+        <Text style={styles.titles}>Description</Text>
         <TextInput
           style={styles.inputStyle}
           value={descriptionText}
-          onChangeText={setDescriptionText}
+          onChangeText={(text) => {
+            setDescriptionText(text);
+            setDescriptionError(''); // Clear error message on change
+          }}
           maxLength={200}
           placeholder="Enter max 200 characters."
           placeholderTextColor="red"
         />
-        {/* <Text style={styles.titles}>Duration</Text>
-        <DurationDropDown /> */}
-        <Text style={styles.titles}>Assign To</Text>
-        {/* <DurationDropDown /> */}
-        <TouchableOpacity
-          onPress={() =>
-            navigation.navigate("Member's page", {
-              setSelectedMember,
-              selectedMember,
-            })
-          }
-        >
-          <Text style={styles.inputStyle}>
-            {selectedMember ? selectedMember : "Enter member's name"}
-          </Text>
-        </TouchableOpacity>
+        {descriptionError ? (
+          <Text style={styles.error}>{descriptionError}</Text>
+        ) : null}
         <View>
-
-
-        <Text style={styles.titles}>Assign To Dropdown</Text>
-          <TouchableOpacity style={styles.dropDownSelector} onPress={selectDropDown}>
-            <Text style={{color:"red"}}>
-              Select User
+          <Text style={styles.titles}>Assign To Dropdown</Text>
+          <TouchableOpacity
+            style={styles.dropDownSelector}
+            onPress={selectDropDown}
+          >
+            <Text style={{ color: 'red' }}>
+              {selectedUser === '' ? 'Select User' : selectedUser?.first_name}
             </Text>
-            {
-              !isDropDownSelected?           
-              <Image source={require("./../../../../assets/dropdown.png")} style={styles.dropDownIcon}/>:
-              <Image source={require("./../../../../assets/dropup.png")} style={styles.dropDownIcon}/>
-            }  
-        </TouchableOpacity>
-                    {
-              isDropDownSelected?
-              <View style={styles.dropDownArea}>
-                <TextInput
-                  style={[styles.inputStyle, { marginTop: 10, marginHorizontal: 5 }]}
-                  value={searchQuery}
-                  onChangeText={(text) => setSearchQuery(text)}
-                  maxLength={200}
-                  placeholder="Search User"
+            {!isDropDownSelected ? (
+              <Image
+                source={require('./../../../../assets/dropdown.png')}
+                style={styles.dropDownIcon}
+              />
+            ) : (
+              <Image
+                source={require('./../../../../assets/dropup.png')}
```

Comment:
please import this image in constant file , don't write the whole in source

Reply:
this needs to be uncommented when backend is fixed that why i kept it this way